app/models/hostgroup.rb

```diff
@@ -195,14 +194,18 @@ def root_pass
 
   # Clone the hostgroup
   def clone(name = "")
-    new = self.deep_clone(:include => [:config_groups, :lookup_values, :hostgroup_classes, :locations, :organizations, :group_parameters],
-                          :except => [:name, :title, :lookup_value_matcher])
+    new = self.deep_clone(:include => [:lookup_values, :hostgroup_classes, :locations, :organizations, :group_parameters],
+                          :except => [:name, :title, :lookup_value_matcher, :config_groups])
```

Comment:
I don't know if this is needed, IIRC it won't clone associations by default - only attributes.  It's causing deprecation warnings in the tests:

<pre>
DEPRECATION WARNING: You're trying to create an attribute `config_groups'. Writing arbitrary attributes on a model is deprecated. Please just use `attr_writer` etc. (called from clone at /home/dcleal/code/foreman/foreman/app/models/hostgroup.rb:197)
</pre>